### PR TITLE
Load in teaser

### PR DESCRIPTION
**Pull request by PhilippDemmelmair**
Implemented that my database content get's loaded into the teaser cards.

### CODE REVIEW

**Review by JessicaLoers — APPROVED** [review]
Well Done 🚀

File: api/teaser/[articleId].js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import connectToMongodb from '../../backend/db/connect';
+import Blogposts from '../../backend/models/Blogposts';
+
+export default async function handler(request, response) {
+	await connectToMongodb();
+
+	const {articleId} = request.query;
+	const post = await Blogposts.findById(articleId);
+
+	switch (request.method) {
```
Comment (JessicaLoers) [inline api/teaser/[articleId].js:10]: Why are you using a switch here?  🤔 `try catch` is more common.

File: src/components/blogposts/BlogTeaser.js [hunk 2]
```diff
@@ -1,23 +1,17 @@
+import {Link, Outlet} from 'react-router-dom';
```
Comment (JessicaLoers) [inline src/components/blogposts/BlogTeaser.js:1]: If you don't use React Routers `Outlet` please remove.🙂

File: src/pages/article.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import {Helmet} from 'react-helmet';
+import {Params, useParams} from 'react-router-dom';
```
Comment (JessicaLoers) [inline src/pages/article.js:2]: Same here. Please remove `Params` from your import.

**Review by JessicaLoers — APPROVED** [review]
There are a few things I would have done differently, but when it works, it works.  👍

File: api/teaser/[articleId].js (hunk 1 above)
Comment (PhilippDemmelmair) [reply api/teaser/[articleId].js:10]: later, there will also be delete method for specific posts, so in advance I chose to use switch. So I don't have to change it later.

File: src/pages/index.js [hunk 4]
```diff
@@ -2,19 +2,39 @@ import {Helmet} from 'react-helmet';
...
+		return (
+			<Layout>
+				<Helmet>
+					<title key="title">My Blog</title>
+					<meta key="description" name="description" content="This is my blog" />
+				</Helmet>
+				{data.data.map(teaser => {
+					// console.log(teaser.title);
+					// console.log(teaser.author);
+					console.log(teaser.published.$timestamp);
```
[the remark this answers is not in the document]
Comment (PhilippDemmelmair) [reply src/pages/index.js]: were already removed in an earlier commit

**Pull request merged**